torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -910,6 +910,11 @@ def _is_packable_linear(match):
         Check if the node is supported for MKLDNN linear.
         """
         linear_node = match.output_node()
+        # mkldnn linear only supports beta=1 and alpha=1
+        # beta_idx=3, alpha_idx=4
+        if linear_node.target == aten.addmm.default:
+            if match.args[3] != 1.0 or match.args[4] != 1.0:
```

Comment:
when beta is 0, it is the case without bias. mkldnn linear still supports it, right?

Reply:
Maybe using `KeywordArg("beta")` like https://github.com/pytorch/pytorch/blob/5f601a41e0a8c91ecf7ca5e4b95d752166ed9093/torch/_inductor/fx_passes/mkldnn_fusion.py#L972. It should be more readable.

Reply:
@jgong5 You are right. I have added the support of beta=0.

Reply:
@leslie-fang-intel Thanks, Leslie. I tried to use `KeywordArg`, but it could not be used by `beta` and `alpha`, as they are optional.

Reply:
@leslie-fang-intel Thanks! Modified to `KeyWordArg`.